core/src/main/java/io/github/fourlastor/game/demo/round/step/SimpleStep.kt

```diff
@@ -8,4 +15,17 @@ abstract class SimpleStep : Step<Unit>() {
     override fun enter(state: GameState, continuation: Consumer<Unit>, cancel: Runnable) {
         enter(state) { continuation.accept(Unit) }
     }
+
+    fun healthDeplete(stage: Stage, assetManager: AssetManager, x: Float, y: Float, damageAmount: Int): Action {
+        // (sheerst) - can we do this without having to pass stage and assetmanager?
```

Comment:
yep, make this its own class and inject them there, so it can have them injected (and we don't need to bring them around like this)